docs/Drawer.md

```diff
@@ -133,7 +133,7 @@ The `items` prop of the `DrawerNavGroup` takes a list of items with the followin
 | title           | The text to show on the first line                                                                             | `string`           | yes      |                                  |    
 
 #### NestedNavItem Object
-The `items` property of the NavItem can be nested to create a tree structure with expand/collapse panels. Nested items take the same properties as `NavItem` with the exception of `icon` (nested items cannot use icons).
+The `items` property of the NavItem can be nested to create a tree structure with expand/collapse panels. Nested items take the same properties as `NavItem` with the exception of `icon` (nested items cannot use icons). Besides, the default `expandIcon` prop is `<ArrowDropUp />`.
```

Comment:
We don't need to call out the default icon here...they'll see it when they use the component. Or if you really want to tell them what it is, you can call it out in the API row above.